### DISCUSSION TITLE
character recognition in code of the project

### DISCUSSION THREAD

**Discussion by Kareemic** [jomjol/AI-on-the-edge-device#2452] [Q&A]
So im interested where in the code is number recognition that reads the value of the number on the image. also how exactly does it all work. im curious because one day i would like to do a project as amazing as this.

**Comment by caco3** [comment]
The project is quite complex, do not use it to get into programming. Search for easier tutorials about number recognition and OCR. There are several ones based on an ESP32-CAM.
